release-notes/v0.4.0.md

```diff
@@ -0,0 +1,34 @@
+manubot version 0.4.0 release notes
+===================================
+
+Includes backwards incompatable changes to the manubot.cite API.
+Major enhancements to the flexibility of citation processing.
+
+Commits
+-------
+
+4701df5 Update release instructions for new workflow
```

Comment:
Should this be a markdown list?

Reply:
since github issues and release notes
leave
single
newlines
alone
this will show up properly.

But would be nice to have them show up properly when rendering `release-notes/v0.4.0.md` in a normal md viewer